les/handler_test.go

```diff
@@ -51,7 +51,8 @@ func TestGetBlockHeadersLes1(t *testing.T) { testGetBlockHeaders(t, 1) }
 func TestGetBlockHeadersLes2(t *testing.T) { testGetBlockHeaders(t, 2) }
 
 func testGetBlockHeaders(t *testing.T, protocol int) {
-	pm := newTestProtocolManagerMust(t, false, downloader.MaxHashFetch+15, nil, nil, nil, ethdb.NewMemDatabase())
+	db := ethdb.NewMemDatabase()
```

Comment:
If you don't use db after calling newTestProtocolManagerMust then there is no need to create it in a separate line.

Reply:
It looks consistent for me

Reply:
Sure, it is consistent, I'm just saying that in the current master branch ethdb.NewMemDatabase() is called in-line and this PR changes that. Not a problem though, this is correct too.